.github/workflows/pr-3-bump-tag.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow is used to convert the version in the `VERSION` file into a valid `git tag` on push to `main`.
+# We use the `VERSION` file to denote the version of the config once a PR is merged.
+name: Bump Tag
+on:
+  push:
+    branches:
+      - 'release-*'
+    paths:
+      - 'VERSION'
+jobs:
+  tag-update:
+    name: Check and Update Tag
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+
+      - name: Existing Tag Check
+        # Check if the tag already exists, if it does, we don't want to move it.
+        id: tag
+        run: |
+          VERSION_TAG=$(git tag -l $(cat VERSION))
```

Comment:
They are separate, unconnected workflows. If we could, we wouldn't have a need for a `VERSION` file at all...maybe this would require a bit more thinking. 
Putting random thoughts down for now...This might be similar to a bit that I do for the prerelease workflow (diff here:  https://github.com/ACCESS-NRI/ACCESS-OM2/pull/35/files#diff-ea3ea8c9932adc7ba8161ceda844fedd43b006848ef1140c050cbd7ea0788a18R7-R37). On merge to `main`, I use the `gh` utility to find the most recent merged pr. This would contain the job (and hopefully the outputs of a job?) that could have that version in it (we do something similar in this PR, actually, see: https://github.com/ACCESS-NRI/access-om2-configs/pull/2/files#diff-a59a7fa546091e4863eebec53b2a6f5e1dd4882500460c9da05f0b8894e08822R104-R111, except that instead of downloading the artifact associated with the workflow, we see the outputs of a given job...)

Reply:
Probably have to put a version in the `metadata.yaml` anyway.